frontend/src/Admin/DTI-Site-Deployer/SiteDeployer.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Button, Card, CardGroup, Loader } from 'semantic-ui-react';
+import { UserContext } from '../../UserProvider/UserProvider';
+import { Member, MembersAPI } from '../../API/MembersAPI';
+import Emitters from '../../EventEmitter/constant-emitters';
+import { backendURL } from '../../environment';
+import styles from './SiteDeployer.module.css';
+import APIWrapper from '../../API/APIWrapper';
+
+const SiteDeployer: React.FC = () => {
+  const userEmail = useContext(UserContext).user?.email;
+
+  const getUser = async (email: string): Promise<Member> => {
+    const mem = await MembersAPI.getMember(email);
+    return mem;
+  };
+
+  const [isLoading, setLoading] = useState(true);
+  const [pullRequests, setPullRequests] = useState<{ body: string }[]>([]);
+
+  const loadPullRequests = () => {
+    APIWrapper.get(`${backendURL}/getIDOLChangesPR`, {}).then((resp) => {
+      if (!resp.data.pr) {
+        setPullRequests([]);
+        setLoading(false);
+        return;
+      }
+      setPullRequests([resp.data.pr]);
+      setLoading(false);
+    });
+  };
+
+  useEffect(() => {
+    if (userEmail) {
+      getUser(userEmail)
+        .then((mem) => {
+          if (mem.role !== 'lead') {
```

Comment:
seems like only leads are able to access this the site deployment page atm

Reply:
Ooh, good catch. I'll fix that rq.